dev/com.ibm.ws.security.social/test/com/ibm/ws/security/social/internal/utils/OpenShiftUserApiUtilsTest.java

```diff
@@ -229,41 +406,54 @@ public void missingUserKeyFromResponse() {
             });
             String errorResponse = "{\"kind\":\"TokenReview\",\"apiVersion\":\"authentication.k8s.io/v1\",\"metadata\":{\"creationTimestamp\":null},\"spec\":{\"token\":\"somebadvalueForAnAccessToken\"},\"status\":{\"user\":{},\"error\":\"[invalid bearer token, token lookup failed]\"}}";
             userApiUtils.modifyExistingResponseToJSON(errorResponse);
-            //assertEquals(returnedString, "{\"username\":\"admin\",\"groups\":}");
-
         } catch (SocialLoginException e) {
-            //nls 
-            verifyException(e, "CWWKS5374E");
-            // fail()
+            verifyException(e, "KUBERNETES_USER_API_RESPONSE_ERROR");
         } catch (Throwable t) {
             outputMgr.failWithThrowable(testName.getMethodName(), t);
         }
     }
 
     @Test
-    public void noUserNameAttributeTest() {
+    public void userNameAttributeNull() {
```

Comment:
Changes in https://github.com/OpenLiberty/open-liberty/pull/9964 should make this scenario impossible.